analysis/import/TokeiImporter/src/test/kotlin/de/maibornwolff/codecharta/importer/tokeiimporter/TokeiImporterTest.kt

```diff
@@ -61,7 +83,7 @@ class TokeiImporterTest {
 
     @Test
     fun `handles path separator correctly`() {
-        val cliResult = executeForOutput("", arrayOf("src/test/resources/tokei_results.json", "--pathSeparator=\\"))
+        val cliResult = executeForOutput("", arrayOf("src/test/resources/tokei_results.json", "--path-separator=\\"))
```

Comment:
Just as a note: independent changes are ideally left out (it's fine to keep them now).

Reply:
That is true. Enabling the unit tests and fixing them should've been it's own PR.

Reply:
Don't we support camelCase as well?

Reply:
Yes, but:
> Fix tests by mostly kebab-casing cli arguments (arguments can be kebab-case AND camelCase. However, the sanitzing only happens when running a real console. This is not the case in those tests. Therefore arguments need to be kebab-cased)